### PR TITLE
Make PrimitiveRenderTypes correctly output long values with tests

### PR DESCRIPTION
**Pull request by omrihq**
This addresses https://github.com/foursquare/fsqio/issues/45

### CODE REVIEW

File: src/resources/io/fsq/ssp/codegen/scala/constants.ssp [hunk 1]
```diff
@@ -8,7 +8,12 @@
 
 object ${baseName}Constants {
   #for (const <- constants)
-  <% val value = const.valueOption.map{v => "val %s: %s = %s".format(const.name, const.renderType.text, v)} %>
+  <% val value = const.valueOption.map{v => {
```
Comment (iantabolt) [inline src/resources/io/fsq/ssp/codegen/scala/constants.ssp]: I'm thinking we should move this higher up, either in `ScalaConst.valueOption` or better `PrimitiveRenderType.renderValue`. Is there ever a time where we _don't_ want the `L`?

**Comment by iantabolt** [comment]
Thanks for this! Can you add a test? I think it would go in `test/thrift/io/fsq/spindle/codegen/runtime/constants/test/gen/constantine.thrift` and `test/jvm/io/fsq/spindle/codegen/runtime/test/ScalaConstTest.scala`.

File: test/jvm/io/fsq/spindle/codegen/runtime/test/ScalaConstTest.scala [hunk 2]
```diff
@@ -9,6 +9,7 @@ class ScalaConstTest {
   @Test
   def testConstantGeneration(): Unit = {
     A.assertEquals(List(1.2, 2.1, 1.1), ConstantineConstants.LISTCONST)
+    A.assertEquals(1000000000, ConstantineConstants.LONG)
```
Comment (iantabolt) [inline test/jvm/io/fsq/spindle/codegen/runtime/test/ScalaConstTest.scala]: This number is in range of integers so it doesn't confirm that the `L` worked. I think one more zero will do it (and then add the `L` so the test compiles)

**Comment by omrihq** [comment]
Good catch, updated

**Comment by mateor** [comment]
Patch was accepted and is live in Fsq.io. Thank you!

fa5c61a688d2c59d496e639726ba5b4d472307f6

**Pull request closed without merging**